AlphabetOrder.java

```diff
@@ -1,9 +1,13 @@
 import java.util.*;
 import java.io.*;
 
-class AlphabetOrder
-{
-	public static void main(String args[])
-	{
-		Scanner in =  new Scanner(System.in);
-		System.out.println("Enter any string: ");
\ No newline at end of file
+class AlphabetOrder {
+	public static void main(String args[]) {
+		Scanner in = new Scanner(System.in);
+		System.out.println("Enter any string: ");
+		String str = in.nextLine();
+		char[] temp = str.toCharArray();
+		Arrays.sort(temp);
```

Comment:
do not use in built method create your own API for this.

Reply:
Done